Source/NSPopUpButton.m

```diff
@@ -232,6 +232,14 @@ - (void) removeItemAtIndex: (NSInteger)index
   [_cell removeItemAtIndex: index];
 
   [self synchronizeTitleAndSelectedItem];
+
+  // TESTPLANT-MAL-09182017: removing causes a multiple sequence eventually
+  //   deselecting any item - after multiple attempts to try to avoid the sequence
+  //   we'll just override the selection at this point...
+  if ([_cell usesItemFromMenu] && ([self selectedItem] == nil) && ([self numberOfItems] > 0))
+    {
+      [self selectItemAtIndex: [self numberOfItems]-1];
+    }
```

Comment:
This is apparently working around a bug in `-synchronizeTitleAndSelectedItem`, as the [docs there](http://gnustep.org/resources/OpenStepSpec/ApplicationKit/Classes/NSPopUpButton.html) say "If there's no selected item, this method selects the first item in the item list and sets the receiver's title to match." but on line 239 the selected item is nil.